test/LayoutTest.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import test from 'tape';
+
+import Layout from '../src/Layout';
+
+test('<Layout>', t => {
+  const wrapper = shallow(<Layout skin="blue">
+      <p>asdf</p>
+    </Layout>);
+
+  t.ok(wrapper.hasClass('sidebar-mini'), 'has (default) sidebar-mini class');
+  t.ok(wrapper.children().hasClass('wrapper'), 'renders child wrapper');
+  t.end()
+});
+
+test('<Layout> skin', t => {
+  const wrapper = shallow(<Layout skin="blue">
+      <p>asdf</p>
+    </Layout>);
+
+  t.ok(wrapper.hasClass('skin-blue'), 'is used');
```

Comment:
I had meant to remove the `skin` definition from the first test, would you like me to do that and keep the skin test as it's own separate test, or move the assertion up and remove the isolated skin test?